doc/changes.md

```diff
@@ -1,5 +1,98 @@
 # Changelog
 
+## Release v0.13.0 (2024/09/05)
+
+[![Digital Object Identifier for PyGMT v0.13.0](https://zenodo.org/badge/DOI/10.5281/zenodo.13679420.svg)](https://doi.org/10.5281/zenodo.13679420)
+
+### Highlights
+
+*
+
+### New Features
+
+* Wrap the GMT API function GMT_Read_Data to read data into GMT data containers ([#3324](https://github.com/GenericMappingTools/pygmt/pull/3324))
+* Wrap GMT's standard data type GMT_IMAGE for images ([#3338](https://github.com/GenericMappingTools/pygmt/pull/3338))
+
+### Enhancements
+
+* pygmt.x2sys_cross: Refactor to use virtualfiles for output tables [BREAKING CHANGE: Dummy times in 3rd and 4th columns now have np.timedelta64 type] ([#3182](https://github.com/GenericMappingTools/pygmt/pull/3182))
```

Comment:
Perhaps we should add a separate category for breaking changes?

Reply:
Would make those changes more / directly visible to users!

Meaning we have to define a new label "breaking changes" and add it to 
https://github.com/GenericMappingTools/pygmt/blob/6dc8eb09fccb7c554ba4a775715533b33dc84d05/.github/release-drafter.yml#L8-L20

Reply:
Common changelog uses a prefix like `**Breaking**` or `**<subsystem> (breaking):**` for highlighting breaking changes, see https://github.com/vweevers/common-changelog?tab=readme-ov-file#244-prefixes. Maybe we could follow that instead of adding a new subsection?

Reply:
Sounds good. Changes in 7d3ac6339f085ad41fa2a1174342a5558dadd46b.